src/pages/events/configure-commerce.md

```diff
@@ -142,14 +142,150 @@ Commerce provides two sources for events: observers and plugins. You must specif
+   │  ◯   │ @adobe/aem-cf-editor-ui-ext-tpl *                           │ Extensibility template for AEM Content Fragment Editor      │ aem/cf-editor/1                        │ action, ui                             │
+   ├──────┼─────────────────────────────────────────────────────────────┼─────────────────────────────────────────────────────────────┼────────────────────────────────────────┼────────────────────────────────────────┤
+   │  ◯   │ @adobe/generator-app-aem-react *                            │ Template for AEM React SPA based on WKND content.           │ N/A                                    │ ui                                     │
+   ├──────┼─────────────────────────────────────────────────────────────┼─────────────────────────────────────────────────────────────┼────────────────────────────────────────┼────────────────────────────────────────┤
+   │  ◯   │ @adobe/aem-cf-admin-ui-ext-tpl *                            │ Extensibility template for AEM Content Fragment Admin       │ aem/cf-console-admin/1                 │ action, ui                             │
+   │      │                                                             │ Console                                                     │                                        │                                        │
+   └──────┴─────────────────────────────────────────────────────────────┴─────────────────────────────────────────────────────────────┴────────────────────────────────────────┴────────────────────────────────────────┘
+   ```
+
+   * Enter the name of the non-web Runtime Action (generic by default).
+
+   ```bash
+   ✔ Installed npm package @adobe/generator-app-events-generic
+   ℹ Running template @adobe/generator-app-events-generic
+   ? We are about to create a new sample action that A generic action that logs the events received from IO Events.
+   How would you like to name this action? generic
+   ```
+
+   * Enter the name of `Event Registration` that will be created (Event Registration Default by default).
+
+   ```bash
+   ? We are about to create a new Event registration.
+   How would you like to name this registration? Customer Event Registration
+   ```
+
+   * Enter the description about what the registration is being created for (Registration for IO Events by default).
+
+   ```bash
+   ? What is this registration being created for? Registration for receiving customer actions
+   ```
+
+   * Choose `Commerce Events` from the list of event providers families.
+
+   ```bash
+   ? Choose from the following provider families ( provider metadata ) Commerce Events
+   ```
+
+   * Choose the `Event Provider` you created in the [Create an Event Provider](./configure-commerce.md#create-an-event-provider) section.
+
+   ```bash
+   ? Choose from below provider for provider metadata: dx_commerce_events team-mercury-sandbox
```

Comment:
Also, it was a part of AC in this task https://jira.corp.adobe.com/browse/CEXT-2856
```
Add context to Merchant ID field - what it is used for. Update screenshot so that "demo" is not our merchant ID field. Change to "TeamMercuryDeveloperSandbox"
```
So, I changed the `demo` values to `team-mercury` in this PR https://github.com/AdobeDocs/commerce-extensibility/pull/218/files